react/components/ListItem.tsx

```diff
@@ -69,10 +72,20 @@ const ListItem: FunctionComponent<Props> = ({
       {/* Quantity Selector */}
       <div className="flex-auto-m mt6-m">
         <div className="dn-m" style={{ width: '70px' }}>
-          <Selector value={item.quantity} onChange={onQuantityChange} />
+          <Selector
+            value={item.quantity}
+            maxValue={MAX_ITEM_QUANTITY}
```

Comment:
Why do you need this as prop? Why it isnt a constant of the component itself?

Reply:
At first I figured this would make it easier to reuse this component if we later wanted to.
But now I discussed with Davi and we'll try to get the API limit and use it to prevent the user from selecting a value greater than the API allows. Since each item can have different limits (I think), this prop will become necessary.